<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyPopUp</class>
 <widget class="QWidget" name="MyPopUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MyPopUp{
	border:none;
background-color: qlineargradient(spread:reflect, x1:1, y1:1, x2:1, y2:0, stop:0.176136 rgba(147, 139, 139, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="popUpGrid" native="true">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>225</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#popUpGrid{
	border: none;
	max-height: 225;
	max-width: 300;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="title">
        <property name="styleSheet">
         <string notr="true">#title{

	color: rgb(44, 44, 44);
	font-size: 20px;
	qproperty-alignment: AlignCenter;
	qproperty-wordWrap: true;
}</string>
        </property>
        <property name="text">
         <string>ERROR!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="horizontalWidget1" native="true">
        <property name="styleSheet">
         <string notr="true">#horizontalWidget1{
	height: 150px;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="messageIcon">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>45</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#messageIcon{
	margin-top: 5px;
	max-width: 40px;
	max-height: 40px;
	image: url(:/images/img/error.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="errorMessage">
           <property name="styleSheet">
            <string notr="true">#errorMessage{
	color: rgb(44, 44, 44);
	font-size: 17px;
	padding-left: 2px;
	padding-right: 2px;
	qproperty-alignment: AlignCenter;
	qproperty-wordWrap: true;
	min-height:100px;
}
</string>
           </property>
           <property name="text">
            <string>There are no blood bags present. Please Insert blood bags</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="horizontalWidget2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="errorButton">
           <property name="styleSheet">
            <string notr="true">#errorButton{
	color: rgb(31, 33, 30);
	border:none;
	outline:none;
	font-size: 20px;
	max-width: 100px;
	min-height: 20px;
	padding: 5px 5px;
	text-align:center;
	subcontrol-position: center;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, 	  stop:0.443182 rgba(98, 116, 146, 255), stop:1 rgba(255, 255, 255, 255));
}

#errorButton:hover{
	background-color: rgb(255,255, 250,50%);
}
</string>
           </property>
           <property name="text">
            <string>Ok</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
